renovate.json

```diff
@@ -42,7 +29,6 @@
 			"groupSlug": "github-actions"
 		}
 	],
-	"allowedPostUpgradeCommands": ["^npm install$", "^cd angular && npm run ng update .*", "^rm package-lock.json$"],
```

Comment:
You removed `"^rm package-lock.json$"` but it is used at @line 16. Does it have any impacts?